tests/integration/conftest.py

```diff
@@ -42,6 +42,13 @@ def cleanup_environment():
                 logging.debug(f"Docker ps before start:{r.stdout}")
         else:
             logging.debug(f"No running containers")
+
+        logging.debug("Pruning Docker networks")
+        run_and_check(
```

Comment:
I'm curios how it works, since it should ask without `--force`.

Reply:
Ok I see, in my env it has stdin on CI it doesn't, let's fix it - https://github.com/ClickHouse/ClickHouse/pull/52095